pkgs/by-name/ha/hachimarupop/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "hachimarupop";
+  version = "unstable-2022-07-11";
+
+  src = fetchFromGitHub {
+    owner = "noriokanisawa";
+    repo = "HachiMaruPop";
+    rev = "67d96c274032f5a2e1d33c1ec53498fde9110079";
+    hash = "sha256-b1moyTVy0hHGu9/LrQ9k6Isd/LYTSxiuqz3BzrYVbXY=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm444 fonts/ttf/*.ttf -t $out/share/fonts/truetype/
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/noriokanisawa/HachiMaruPop";
+    description = "A cute, Japanese font";
+    license = lib.licenses.ofl;
+    maintainers = with lib.maintainers; [ AndersonTorres ];
```

Comment:
Quick question, is this intended? As this is a new package PR, I think you should set yourself as the maintainer

Reply:
@WolfangAukang i see.
what happened was, i was seeing a merge conflict here listing this particular maintainer (and altered package name using dashes), figured someone else might have edited my PR (not sure how), and rolled with it, accepting what i considered to be their changes.
i'm open to changing it again tho.

Reply:
No problem, that makes it clear. Checking it thoroughly, someone packaged it and got merged before yours (https://github.com/NixOS/nixpkgs/pull/267922).